### ISSUE TITLE
Intel I225-V wired ethernet not getting initialized by igc kernel module

### ISSUE THREAD

**Issue by garza** [holoiso-eol/holoiso#596]
## Your Issue:
```
wired ethernet kernel mod not working: Intel I225-V - igc

the wired ethernet is working correctly when booting with the holoiso kernel (5.18.1-arch1_testHoloISO_20220606.1811)

modprobe igc results in current 5.13.0-valve31-1-neptune:
[13682.215634] Intel(R) 2.5G Ethernet Linux Driver
[13682.215637] Copyright(c) 2018 Intel Corporation.
[13682.215773] igc 0000:01:00.0: no suspend buffer for PTM
[13682.243729] igc: probe of 0000:01:00.0 failed with error -2

lspci -v:
01:00.0 Ethernet controller: Intel Corporation Ethernet Controller I225-V (rev 03)
        DeviceName: Onboard LAN Brodcom
        Subsystem: Intel Corporation Device 0000
        Flags: fast devsel, IRQ 24, IOMMU group 10
        Memory at fcc00000 (32-bit, non-prefetchable) [size=1M]
        Memory at fcd00000 (32-bit, non-prefetchable) [size=16K]
        Capabilities: [40] Power Management version 3
        Capabilities: [50] MSI: Enable- Count=1/1 Maskable+ 64bit+
        Capabilities: [70] MSI-X: Enable- Count=5 Masked-
        Capabilities: [a0] Express Endpoint, MSI 00
        Capabilities: [100] Advanced Error Reporting
        Capabilities: [140] Device Serial Number 58-47-ca-ff-ff-70-14-10
        Capabilities: [1c0] Latency Tolerance Reporting
        Capabilities: [1f0] Precision Time Measurement
        Capabilities: [1e0] L1 PM Substates
        Kernel modules: igc

WiFi and BT are working great (mt7921e)

```

## Your device's full specs
```
CPU: AMD Ryzen 7 5800H
GPU: AMD integrated Renoir/Cezanne
uname -a:
Linux 5.13.0-valve31-1-neptune #1 SMP PREEMPT Tue, 08 Nov 2022 22:43:07 +0000 x86_64 GNU/Linux
Minisforum UM580 pc
```

## Your OS version (`cat /etc/os-release | grep VARIANT_ID | cut -d '"' -f 2`):
```
snapshot20221029.1010
```

## (Only for SteamOS Session loading issues) Have you tried launching `gamescope-session` from terminal? If yes, provide a log.
```

```

## Your installation type:
```
deckperience
```

## Have you updated your installation recently?
```
yes,  # pacman -Syu
:: Synchronizing package databases...
 holoiso is up to date
 holostaging is up to date
 jupiter-main                                            86.0 KiB   174 KiB/s 00:00 [################################################] 100%
 holo-main is up to date
 core-main is up to date
 extra-main is up to date
 community-main is up to date
 multilib-main is up to date
 andontie-aur is up to date
:: Starting full system upgrade...
warning: jupiter-hw-support: ignoring package upgrade (20221010.1-1 => 20221109.3-1)
 there is nothing to do
```

**Comment by IM1903** [comment]
I have a similar problem but mine's even worse. Installer just stopped at `igc 0000:07:00.0: no suspend buffer for PTM ` and nothing more.

**Comment by garza** [comment]
this has been fixed by the 6.0.9-valve1-2-neptune kernel in the latest release, see the update instructions in telegram
